tutorials/nnvm/from_darknet.py

```diff
@@ -55,7 +55,7 @@
 # Download and Load darknet library
 # ---------------------------------
 
-darknet_lib = 'libdarknet.so'
+darknet_lib = 'libdarknet2.0.so'
```

Comment:
is it possible to provide lib for Mac and windows?

Reply:
I think it's better to give steps to generate this lib from source instead (Many platforms and lot of compilers).

Reply:
Binaries can be build from sources and can be imported to run the model. For demo purpose, i will update the binaries for Mac and Windows.

Reply:
Where can we find the build scripts for compiling this library?